content/docs/user-guide/external-dependencies.md

```diff
@@ -54,11 +54,13 @@ $ dvc run -n download_file
 
 ```dvc
 $ dvc run -n download_file
-          -d ssh://user@example.com:/home/shared/data.txt \
+          -d ssh://user@example.com/home/shared/data.txt \
           -o data.txt \
-          scp user@example.com:/home/shared/data.txt data.txt
+          scp user@example.com/home/shared/data.txt data.txt
 ```
 
+> Please notice `/home/...` is an absolute path from the remote system's root.
```

Comment:
instead of notes - let's do /absolute/root/path/from... ? (I don't like notes - there should be a very strong reason to use them).

Reply:
Good idea, done. But `/absolute/path/from/sftp/root/which/istnt/always/system/root` would be excessive. Even without the "which..." part, I think. So I kept the note for now to warn about the common SFTP root confusion (see https://github.com/iterative/dvc.org/pull/1649#pullrequestreview-459591916 above).

I agree it's best to avoid notes (like avoiding comments in code) but at the same time this is exactly what ⚠️ side-notes (about external tools) are for, and we have plenty of them.

Reply:
p.s. I did remove the notes completely from HDFS sections.

Reply:
UPDATE: With the note (only on SSH) now being

> ⚠️ Please notice `/abs/path/to/...` is a path relative to the SFTP root
> (typically the system root, in which case it's an absolute path).

it may be confusing again since the URL starts with `abs`/`absolute` but the note calls it "relative"... So I guess just `/path/from/sftp/root` + further simplify or remove the note may be better after all...

Reply:
OK... Changed again, using longer `/path/from/sftp/root`... now.